vuejs/nft-basic/src/App.vue

```diff
@@ -214,10 +214,9 @@ import axios from 'axios';
 import ipfsClient from './ipfs';
```

Comment:
Shouldn't we remove the `ipfsClient` too?

Reply:
Removed all instances of IPFS from code